src/include/migraphx/streamutils.hpp

```diff
@@ -27,6 +27,7 @@
 #include <ostream>
 #include <algorithm>
 #include <migraphx/rank.hpp>
+#include <migraphx/value.hpp>
```

Comment:
Actually instead of adding the value.hpp header here. Can we just add a `operator<<` to the `value::binary` class?

Reply:
`value` class already has `friend operator <<()`.  For the  binary it prints entire binary blob.  in `streamutils`, If i  remove `rank<3>` method then, i think it would still match with `rank<2>` method as a container.

Reply:
I am saying to add `operator<<` to `value::binary` class not to the `value` class.

Reply:
Done.